CodingDojo/server/src/main/java/com/codenjoy/dojo/services/PlayerServiceImpl.java

```diff
@@ -133,9 +133,9 @@ public Player register(String id, String game, String room, String ip) {
             {
                 save.setCallbackUrl(ip);
             } else {
-                save = new PlayerSave(id, ip, game, room, 0, null);
+                save = new PlayerSave(id, save.getTeamId(), ip, game, room, 0, null);
```

Comment:
тут save == PlayerSave.NULL так и задумано? Может быть Взять сразу DEFAULT_TEAM

Reply:
not really
there're two more conditions
`save != PlayerSave.NULL 
                    && game.equals(save.getGame())
                    && room.equals(save.getRoom())`
it's not a problem in any way
if there's value - than we take it
if there's no value - than it would be DEFAULT_TEAM_ID

Reply:
Есть ли тесты на это поведение? 
if there's value - than we take it
if there's no value - than it would be DEFAULT_TEAM_ID

Reply:
do you suggest to test getter?

Reply:
or I should test the constructor?

Reply:
Речь о PlayerServiceImpl.register() методе.

Reply:
implemented `testRegister_passTeamIdFromSave` & `testRegister_passDefaultTeamIdFromSave` & `testRegister_passDefaultTeamIdFromNullSave` test cases
where capture PlayerSave and check its value whether it's provided or default one